packages/channel-client/tests/client-provider-wallet.test.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
+
+import {IFrameChannelProviderInterface} from '@statechannels/iframe-channel-provider';
+
+jest.setTimeout(30000);
+require('@statechannels/iframe-channel-provider');
```

Comment:
?

Reply:
This is the standard way we expect apps to load the wallet?